int8 box_type

Point point1
Point point2
Point point3
Point point4

string class_id

float32 probability